Description: without a config file, cabal doesn't try to update the cache over internet
 This has been taken from the old 0.5.1 version of the package
Author: Gianfranco Costamagna <locutusofborg@debian.org>
Forwarded: not-needed

--- a/Makefile
+++ b/Makefile
@@ -8,7 +8,7 @@ else
 endif
 NCPUS=$(shell grep -c processor /proc/cpuinfo)
 PARALLEL=$(shell which parallel)
-CABAL=cabal --config-file=./cabal.config
+CABAL=cabal --config-file=/dev/null
 GEN=$(CABAL) new-run -j$(NCPUS) exe:xcffibgen --
 
 # you should have xcb-proto installed to run this
